WIP-SQL/what-is-a-database-server.md

```diff
@@ -0,0 +1,61 @@
+# What is a database server?
+author: SebaRaba
+
+levels:
+
+  - beginner
+
+  - basic
+
+type: normal
+
+category: must-know
+
+links:
+
+  - '[More on database servers](http://ecomputernotes.com/fundamental/what-is-a-database/what-is-a-database-server)'
+
+---
+## Content
+
+A database server can be the *hardware* part of a database. A computer dedicated to perform database storage an retrieval. Such a hardware runs the database software  application (DBMS - database management system). It receives requests from the **client machines** and searches the collection for the **requested records**.
+
+As a software, it means the backend (also called instance) part of an database application. The backend handles tasks such as data analisys, storage, data manipulation and other non-user specific tasks. In general, database management systems provide database-server functionality. There are some DBMS[1] (such as MySQL) which rely exclusively on the *client-server model*[2].
```

Comment:
"As a software, it means the backend (also called instance) part of an database application." -> "When we're referring to the software, we're talking about the program that receives and interprets requests. These requests are called **Queries** which are requests to store, retrieve, or change data in the database."
Database servers can:
* Read data
* Store data
* Analyse data
* Validate data against a Schema

Remove this bit:
"The backend handles tasks such as data analisys, storage, data manipulation and other non-user specific tasks. In general, database management systems provide database-server functionality. There are some DBMS[1] (such as MySQL) which rely exclusively on the *client-server model*[2]."